src/app/paging/paging-virtual.component.scss

```diff
@@ -0,0 +1,7 @@
+.custom-loading-indicator {
+  position: sticky;
+  top: 0;
+  height: 50px;
```

Comment:
Not needed:

```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/69795395/135219724-5e9c70d4-d8d5-43de-bc7e-7a1239dc5c91.png)
If I remove the height then the loading text and table content are overlapping.

Reply:
Wasn't the case during my attempts, but makes sense.